nms/app/fbcnms-projects/magmalte/app/views/equipment/GatewayDetailConfigEdit.js

```diff
@@ -0,0 +1,758 @@
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow strict-local
+ * @format
+ */
+import type {
+  enodeb_serials,
+  gateway_device,
+  gateway_epc_configs,
+  gateway_logging_configs,
+  gateway_ran_configs,
+  lte_gateway,
+  magmad_gateway_configs,
+  package_type,
+} from '@fbcnms/magma-api';
+
+import Button from '@material-ui/core/Button';
+import Dialog from '@material-ui/core/Dialog';
+import DialogActions from '@material-ui/core/DialogActions';
+import DialogContent from '@material-ui/core/DialogContent';
+import DialogTitle from '@material-ui/core/DialogTitle';
+import FormLabel from '@material-ui/core/FormLabel';
+import Link from '@material-ui/core/Link';
+import List from '@material-ui/core/List';
+import MagmaV1API from '@fbcnms/magma-api/client/WebClient';
+import OutlinedInput from '@material-ui/core/OutlinedInput';
+import Paper from '@material-ui/core/Paper';
+import React from 'react';
+import Switch from '@material-ui/core/Switch';
+import Tab from '@material-ui/core/Tab';
+import Tabs from '@material-ui/core/Tabs';
+import Text from '@fbcnms/ui/components/design-system/Text';
+import nullthrows from '@fbcnms/util/nullthrows';
+
+import {AltFormField} from '../../components/FormField';
+import {colors, typography} from '../../theme/default';
+import {makeStyles} from '@material-ui/styles';
+import {useEnqueueSnackbar} from '@fbcnms/ui/hooks/useSnackbar';
+import {useRouter} from '@fbcnms/ui/hooks';
+import {useState} from 'react';
+
+const GATEWAY_TITLE = 'Gateway';
+const RAN_TITLE = 'Ran';
+const AGGREGATION_TITLE = 'Aggregation';
+const EPC_TITLE = 'Epc';
+const DEFAULT_GATEWAY_CONFIG = {
+  cellular: {
+    epc: {
+      ip_block: '192.168.128.0/24',
+      nat_enabled: true,
+      dns_primary: '',
+      dns_secondary: '',
+    },
+    ran: {
+      pci: 260,
+      transmit_enabled: true,
+    },
+  },
+  connected_enodeb_serials: [],
+  description: '',
+  device: {
+    hardware_id: '',
+    key: {
+      key_type: 'ECHO',
+    },
+  },
+  id: '',
+  magmad: {
+    autoupgrade_enabled: true,
+    autoupgrade_poll_interval: 60,
+    checkin_interval: 60,
+    checkin_timeout: 30,
+    dynamic_services: [],
+  },
+  name: '',
+  status: {
+    platform_info: {
+      packages: [
+        {
+          version: '',
+        },
+      ],
+    },
+  },
+  tier: 'default',
+};
+
+const useStyles = makeStyles(_ => ({
+  topBar: {
+    backgroundColor: colors.primary.mirage,
+    padding: '20px 40px 20px 40px',
+    color: colors.primary.white,
+  },
+
+  appBarBtn: {
+    color: colors.primary.white,
+    background: colors.primary.comet,
+    fontFamily: typography.button.fontFamily,
+    fontWeight: typography.button.fontWeight,
+    fontSize: typography.button.fontSize,
+    lineHeight: typography.button.lineHeight,
+    letterSpacing: typography.button.letterSpacing,
+
+    '&:hover': {
+      background: colors.primary.mirage,
+    },
+  },
+  input: {
```

Comment:
@ekfuhrmann is making changes to global dialog settings, might be useful to make those associated changes here.